src/electron/ipc/ipc-channel-names.ts

```diff
@@ -1,5 +1,22 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-export const IPC_MAIN_WINDOW_INITIALIZED_CHANNEL_NAME = '60dc5587-7f98-4216-92d5-7026875b2ee4';
 export const IPC_MESSAGE_CHANNEL_NAME = '9a436d48-61e9-4c34-849d-3c15ef06f2c9';
```

Comment:
P2: might be better to group the event names

something like:
export var ipcEvents = {
   RENDERER_GET_APP_VERSION_CHANNEL_NAME : '',
   ...
}

Reply:
never mind. i see all the event names here are for ipc from renderer

Reply:
I'll reserve this for a separate PR